django-rgd-fmv/rgd_fmv/urls.py

```diff
@@ -4,7 +4,8 @@
 from rgd_fmv.rest import viewsets
 
 router = SimpleRouter(trailing_slash=False)
-router.register(r'api/rgd_fmv', viewsets.FMVViewSet)
+router.register(r'api/rgd_fmv', viewsets.FMVMetaViewSet, basename='fmv-meta')
+router.register(r'api/rgd_fmv/model', viewsets.FMVViewSet, basename='fmv')
```

Comment:
Probably a better name of this but I'm fine with this

Reply:
I can't think of a better one, seems fine to me :shrug: